src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -5,26 +5,49 @@
 import com.google.cloud.datastore.Entity;
 import com.google.cloud.datastore.FullEntity;
 import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Key;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.*;
+import java.io.*;
```

Comment:
[3.3.1 No wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports). I think you need:

```suggestion
import java.io.IOException;
```